shell/browser/api/electron_api_web_frame_main.cc

```diff
@@ -72,7 +72,17 @@ WebFrameMain* WebFrameMain::FromFrameTreeNodeId(int frame_tree_node_id) {
 
 // static
 WebFrameMain* WebFrameMain::FromRenderFrameHost(content::RenderFrameHost* rfh) {
-  return rfh ? FromFrameTreeNodeId(rfh->GetFrameTreeNodeId()) : nullptr;
+  if (!rfh)
+    return nullptr;
+
+  // TODO(codebytere): remove after refactoring away from FrameTreeNodeId as map
+  // key.
+  auto* ftn =
```

Comment:
No - at the moment it doesn't since it was written for the assumption that wouldn't happen. I do think we can use `GlobalRenderFrameHostToken` instead; i can refactor it in this PR but i wasn't sure how much time it'd take to look into it more so I thought i'd mitigate a hard crash before doing so. Do you have a preference?

Reply:
Lets do it as follow-up, we might want to change to track ftn directly rather than the underlying rfh so this needs further discussion.